<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>781</width>
      <height>410</height>
     </rect>
    </property>
    <widget class="QWidget" name="page"/>
    <widget class="QWidget" name="commodityPage">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>781</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>商品管理</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QToolBox" name="toolBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>781</width>
        <height>381</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::WinPanel</enum>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="newCommodity">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>777</width>
         <height>325</height>
        </rect>
       </property>
       <attribute name="label">
        <string>新品入库</string>
       </attribute>
       <widget class="QComboBox" name="cmbNewCategory">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>20</y>
          <width>72</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="labNewCategory">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>20</y>
          <width>41</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>类别：</string>
        </property>
       </widget>
       <widget class="QLabel" name="labNewName">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>50</y>
          <width>41</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>名称：</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="leNewName">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>50</y>
          <width>211</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="labNewInPrice">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>80</y>
          <width>41</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>进价：</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="leNewInPrice">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>80</y>
          <width>61</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="labNewPriceFlag1">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>80</y>
          <width>21</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>￥</string>
        </property>
       </widget>
       <widget class="QLabel" name="labNewOutPrice">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>80</y>
          <width>41</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>售价：</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="leNewOutPrice">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>80</y>
          <width>51</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="labNewPriceFlag2">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>80</y>
          <width>21</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>￥</string>
        </property>
       </widget>
       <widget class="QLabel" name="labNewNumber">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>110</y>
          <width>41</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>数量：</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="spbNewAmount">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>110</y>
          <width>81</width>
          <height>22</height>
         </rect>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="value">
         <number>1</number>
        </property>
       </widget>
       <widget class="QLabel" name="labNewPicture">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>150</y>
          <width>54</width>
          <height>12</height>
         </rect>
        </property>
        <property name="text">
         <string>商品图片：</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnAdd">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>190</y>
          <width>80</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>入库</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnClear">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>230</y>
          <width>80</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>清仓</string>
        </property>
       </widget>
       <widget class="QLabel" name="labNewPixmap">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>150</y>
          <width>151</width>
          <height>151</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="btnUpload">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>130</y>
          <width>80</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>上传...</string>
        </property>
       </widget>
       <widget class="QTableView" name="tableView">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>20</y>
          <width>401</width>
          <height>291</height>
         </rect>
        </property>
        <attribute name="horizontalHeaderMinimumSectionSize">
         <number>25</number>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>120</number>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
       </widget>
      </widget>
      <widget class="QWidget" name="orderCommdity">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>777</width>
         <height>325</height>
        </rect>
       </property>
       <attribute name="label">
        <string>预售订单</string>
       </attribute>
       <widget class="QLabel" name="labPreCategory">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>41</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>类别：</string>
        </property>
       </widget>
       <widget class="QComboBox" name="cmbPreCategory">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>20</y>
          <width>72</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="labPreName">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>41</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>名称：</string>
        </property>
       </widget>
       <widget class="QComboBox" name="cmbPreName">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>50</y>
          <width>211</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="labPreSinglePrice">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>90</y>
          <width>41</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>单价：</string>
        </property>
       </widget>
       <widget class="QLabel" name="labPreSinglePriceNum">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>90</y>
          <width>54</width>
          <height>16</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLabel" name="labPrePriceFlag1">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>90</y>
          <width>21</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>￥</string>
        </property>
       </widget>
       <widget class="QLabel" name="labPreAmount">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>90</y>
          <width>41</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>库存：</string>
        </property>
       </widget>
       <widget class="QLabel" name="labPreAmountNum">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>90</y>
          <width>51</width>
          <height>16</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLabel" name="labPreAmountFlag">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>90</y>
          <width>21</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>件</string>
        </property>
       </widget>
       <widget class="QLabel" name="labPreNumber">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>120</y>
          <width>41</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>数量：</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="spbPreNumber">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>120</y>
          <width>71</width>
          <height>22</height>
         </rect>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="value">
         <number>1</number>
        </property>
       </widget>
       <widget class="QLabel" name="labPreTotalPrice">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>120</y>
          <width>41</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>总价：</string>
        </property>
       </widget>
       <widget class="QLabel" name="labPreTotalPriceNum">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>120</y>
          <width>51</width>
          <height>16</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLabel" name="labPriceFlag2_2">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>120</y>
          <width>21</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>￥</string>
        </property>
       </widget>
       <widget class="QLabel" name="labPrePixmap">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>160</y>
          <width>151</width>
          <height>151</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="btnSell">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>190</y>
          <width>80</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>出售</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnOrder">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>220</y>
          <width>80</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>下单</string>
        </property>
       </widget>
       <widget class="QListWidget" name="listWidget">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>20</y>
          <width>441</width>
          <height>311</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
